sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+_Font Màgica_ gently sounds within a technics and phenomenology of material transformation. Neither a work _of_ traditionally-constructed originality nor one _about_ appropriation, it is a lensing apparatus, a technical-sonic system that is _complicit with_ Chopin and solar flares and Izabella's visuals and my signal processing algorithms. Though slightly different each time due to random signal changes, _Font Màgica_ is not an "open" work so much as an obsessed one. It is obsessed with its own material closing in on itself via signal processing. Chopin's _Raindrop_ appears everywhere, replicated at all levels as both a structuring force as well as a microsonic substrate. _Font Màgica_ aims for a form of systemic closure and inner self-similarity; a closure that paradoxically "realizes openness in its radical sense: not as openness toward the possibility of contingencies from the outside, but as a 'being opened' by the contingent materials that form the work" (Negarestani 3).
+
+Algorithms paint an artwork into a corner. They limit possibilities rather than opening a work up to them. A "rigorous and twisted mode of closure," algorithms allow a work to close in on itself and its materials (2). Too often algorithmic art is couched in a rhetoric of power and control that demands a distinctly perspectival position, like the abstract, theoretical, architectural view from above in De Certeau's reading of the city from the top floor a skyscraper (92-3). For me, algorithms are not about the control afforded by such an over-arching perspective on a system, nor are they simply a Cageian opening up of the work to the contingencies of a constructed "nature" distinct from the technics used to register it. I use algorithms in my work specifically in order to have my view of the aesthetic terrain partially obscured, to find localized energies that emerge not from the abstract adherence to rules, nor openness to chance, but from a kind of obsessive practice that allows a creative system to start feeding on itself, to "uncover itself as the field of experimentation of its contingent materials as a conspiracy plotted by anonymous materials" (Negarestani 4).
```

Comment:
This gets quite precisely at why (despite my own interest in computer-mediated artistic practice) I find so much computer-mediated art uninspiring.

To simply use the computer as a means of deterministic or random production, as might have been done in the pre-computer age by other tools (precise rules for sonnets, or the actions of dice) is to be, in my opinion, largely uninterested in its full potential. 

More is required - as you say, a means of allowing "a creative system to start feeding on itself" - to be a vector for possibilities the artist did not expect, which the artist is then compelled to respond to.